templates/docker-compose/docker-compose.prod.yml.dist

```
@@ -41,6 +41,33 @@ services:
             - db_password
         command: 'bash -c "service cron start && tail -f /var/log/cron.log"'
 
+    supportpal_websockets:
+        container_name: '${WS_SERVICE_NAME}'
+        image: 'public.ecr.aws/supportpal/helpdesk:${APP_VERSION}-sockets-buster'
```

Image is currently unpublished so I think `-ws-buster` is a better suffix
```suggestion
        image: 'public.ecr.aws/supportpal/helpdesk:${APP_VERSION}-ws-buster'
```